<launch>
    <node name="GetObject_client" pkg="robot_arm_control" type="GetObject_client" output="screen">
        <param name="x" type="int"  value="0"/>
        <param name="y" type="int"  value="23"/>
        <param name="z" type="int"  value="3"/>
    </node>
    
</launch>
